ftw/redirector/browser/excel_views.py

```diff
@@ -0,0 +1,108 @@
+from ftw.redirector import _
+from ftw.redirector.config import RulesValidator
+from ftw.redirector.excel import create_rules_excel
+from ftw.redirector.excel import load_rules_from_excel
+from ftw.redirector.excel import RULES_START_ROW
+from ftw.redirector.interfaces import IRedirectConfig
+from ftw.redirector.interfaces import IRedirectorLayer
+from plone.directives import form
+from plone.formwidget.namedfile.widget import NamedFileWidget
+from plone.namedfile.field import NamedBlobFile
+from plone.namedfile.interfaces import INamedFileField
+from Products.Archetypes.utils import contentDispositionHeader
+from Products.Five.browser import BrowserView
+from Products.statusmessages.interfaces import IStatusMessage
+from StringIO import StringIO
+from z3c.form import button
+from z3c.form.interfaces import IFieldWidget
+from z3c.form.validator import WidgetValidatorDiscriminators
+from z3c.form.widget import FieldWidget
+from zope.component import adapter
+from zope.component import provideAdapter
+from zope.interface import implementer
+from zope.interface import Invalid
+
+
+class ExportRedirectConfigView(BrowserView):
+
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
```

Comment:
:scissors: the `__init__` does the same as the one inherited from `BrowserView`, it can safely be removed.